docs/src/images/spectrum.svg

```diff
@@ -0,0 +1,18 @@
+<svg width="126" height="20" viewBox="0 0 126 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0)">
```

Comment:
Because this is an SVG file, ESLint cannot parse it. Even if it was able to, it's not like we could do something about it 😅 In this particular case I managed to avoid the clip path, though 🎉

Reply:
ah, yes :D did not notice :D